modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1826,19 +1826,12 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
         }
     }
 
-    private static String getAccepts(OpenAPI openAPIArg, Operation operation) {
+    private static String[] getAccepts(OpenAPI openAPIArg, Operation operation) {
         final Set<String> producesInfo = getProducesInfo(openAPIArg, operation);
         if (producesInfo != null && !producesInfo.isEmpty()) {
-            StringBuilder sb = new StringBuilder();
-            for (String produce : producesInfo) {
-                if (sb.length() > 0) {
-                    sb.append(",");
-                }
-                sb.append(produce);
-            }
-            return sb.toString();
+            return new LinkedHashSet<>(producesInfo).toArray(new String[] {});
```

Comment:
Why not directly producesInfo.toArray?

Reply:
I did it to guarantee the sort order as I saw the output for the `produces` field was output in another order when generating all the scripts. But it seems like this is not the case (anymore). So I have updated the code to follow your suggestion. :)